src/plone/restapi/tests/test_services_querystringsearch.py

```diff
@@ -319,3 +319,46 @@ def test_querystringsearch_sort(self):
             response.json()["items"][-1]["@id"],
             f"{self.portal.absolute_url()}/testdocument9",
         )
+
+    def test_querystringsearch__invalid_input(self):
```

Comment:
Maybe it's better for each assert to be a separate test. It may be easier to identify the error in case of failure.

Reply:
OTHO, the failure report tells you the line